.gitignore

```diff
@@ -4,3 +4,4 @@ tmp/
 #System Files
 .DS_Store
 Thumbs.db
+.idea
```

Comment:
In general, our practice is to not include editor-specific exclusions in our `.gitignore` files, since we will never be exhaustive about it. These exclusions should be in your `~/.gitignore`. TBH, the same can be said, in my opinion about system files as well.

Reply:
OK i will remove this line